test/e2e/README.md

```diff
@@ -0,0 +1,64 @@
+# end-to-end testing
+
+It is possible to run the SIG Storage-managed end-to-end tests provided by upstream Kubernetes to validate the correctness of the DigitalOcean CSI driver. This document describes how the end-to-end are structured and how they can be performed using the files in this directory.
+
+## Introduction
+
+The Kubernetes project maintains a set of end-to-end tests that are used to validate the correctness of the upstream development. Since Kubernetes 1.14, these tests can also be consumed externally by provisioning a Kubernetes cluster to run the tests against.
+
+This also holds for the storage-related tests but comes with a caveat: the CSI specification gives a lot of flexibility as to which features may be implemented by a drivers, with the implication that not all drivers may implement all functionality that is being verified by the end-to-end tests. Fortunately, the external storage tests [provide a mechanism](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/external) to declare the set of features implemented by a driver-under-test, and the testing framework takes this information into account to selectively choose tests during runtime.
+
+Our end-to-end testing integration customizes the test configuration for DigitalOcean's CSI driver and provides tooling to easily perform the testing.
+
```

Comment:
Can we add a short section describing the `testdrivers` and a link to https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/testsuites/testdriver.go#L135-L156 as a quick reference to what capabilities can be added.

Reply:
Good call. A pushed a commit that amends the README by a new section detailing the testdriver file. Let me know what you think about it.